doc/faq.rst

```diff
@@ -61,6 +61,92 @@ reporting a bug, which should look something like this::
     skcuda:        Not found
 
 
+Why shouldn't I "pickle" my MNE-Python objects for later use?
```

Comment:
yes I agree with @Eric89GXL -- maybe rephrase: "why something atrocious is going to happen if you consider to use pickling !!!"

![gif-keyboard-17370554860503284834](https://cloud.githubusercontent.com/assets/1908618/13843104/0643b118-ec32-11e5-917e-1851cfed2fe2.gif)